CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ if (MSVC)
 
     # Use statically linked runtime
     set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded$<$<CONFIG:Debug>:Debug>)
+
+    # Make directory that 'Build' in Visual Studio would, so we can move BotConfig.txt
+    file(MAKE_DIRECTORY ${CMAKE_SOURCE_DIR}/build/bin/Debug)
+    file(COPY_FILE ${CMAKE_SOURCE_DIR}/bin/BotConfig.txt ${CMAKE_SOURCE_DIR}/build/bin/Debug/BotConfig.txt)
```

Comment:
The problem with this config is that we move it only at 'project configuration' step, i.e. when build routine was initialized first time. As a result, one needs to know that actually he has two versions of config, but only one of them used in the debug mode. That was solved previously by explicit failure on binary start which notified developer that he needs config of some sort.

I would suggest to solve this problem in a different way if possible:
- Rename original config to BotConfig.template.txt (or BotConfig.example.txt). This step is needed to say explicitly that we do not use it anyhow.
- Rename the bin folder in CommandCenter project to config (to be explicit).
- Copy it during configuration time somewhere inside the build folder.
- Use the same config file for release and debug mode.

Reply:
`Use the same config file for release and debug mode.`
I'll have to play with this some.  Seems Visual Studio creates up to 4 folders (Release, Debug, MinSizeRel, RelWithDebInfo).  
One thought was to create a symlink in each folder, but then we are stuck with target_link name.

Reply:
I think the proper way to solve this issue would introduction of '-c, --config' option with location of the config. Looks much simpler, not requires copying/linking and could help in other scenarous.